.github/workflows/sketcher-builder.yml

```diff
@@ -34,10 +34,10 @@ jobs:
           echo PATH=$PATH >> $GITHUB_ENV
 
       - name: Install Python packages
-        run: python3 -m pip install ninja clang-format==14.0.6
+        run: python3 -m pip install cmake==3.30.0 ninja clang-format==14.0.6
```

Comment:
The runners already have CMake 3.31 on them, so you don't need this (and I'm not sure if it's contributing to the debug build failure).  I'm surprised that merging my branch in didn't revert this change.

Reply:
Done.